PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -12,12 +12,20 @@
   , 'inclusiveSecondaryVertexFinderFilteredNegativeTagInfos'
     # new candidate-based framework (supported with RECO/AOD/MiniAOD)
   , 'pfImpactParameterTagInfos'
+  , 'pfImpactParameterTagInfosAK8'
+  , 'pfImpactParameterTagInfosCA15'
   , 'pfSecondaryVertexTagInfos'
   , 'pfSecondaryVertexNegativeTagInfos'
   , 'pfInclusiveSecondaryVertexFinderTagInfos'
+  , 'pfInclusiveSecondaryVertexFinderTagInfosAK8'
```

Comment:
I hope the answer to why we would want to enlarge the dR cone is obvious. We are dealing with fat jets whose subjets or SV can be separated by more than the typical dR requirements used in the b-tagging code.

Also note that PAT did not and still does not modify any of the b-tagging default cuts several of which are of the dR type. Here we have an example of something that was specifically trained for AK8 and CA15 jets and requires "non-standard" cuts so that's why the corresponding TagInfos have the jet identifiers in their names. However, as I mentioned when I sent you the first draft of these taggers, none of the names are set in stone and can be modified. In principle, yes, maybe we can have R8 and R15 instead of AK8 and CA15. None if this is in the standard Reco so we can update this 10 times over. However, I think it is better to have a fixed set of reference config files then to allow PAT to modify everything on the fly.